pkg/bloomgateway/bloomgateway.go

```diff
@@ -80,6 +80,11 @@ const (
 	metricsSubsystem = "bloom_gateway"
 )
 
+var (
+	// responsesPool pooling array of v1.Output [64, 128, 256, ..., 65536]
+	responsesPool = queue.NewSlicePool[v1.Output](1<<6, 1<<16, 2)
```

Comment:
It's easier to understand the number in decimal form. 
```suggestion
	responsesPool = queue.NewSlicePool[v1.Output](64, 65536, 2)
```

Reply:
Happy to change it, but note that we usually declare pools with that notation:
https://github.com/grafana/loki/blob/c405db43179c58e6aef18a8aed8129fb5b83c5a2/pkg/chunkenc/pool.go#L49-L60